server/app/api/blueprints/foo.py

```diff
@@ -0,0 +1,111 @@
+from flask import Blueprint, jsonify, request
+from sqlalchemy import exc
+from app import db
+from app.api.models.foo import Foo
+
+foo_blueprint = Blueprint('foo', __name__)
+
+
+@foo_blueprint.route('/api/foo', methods=['POST'], strict_slashes=False)
```

Comment:
I FORGOT ALL ABOUT THE STRICT SLASHES

Thank you for fixing this 😁 I had to do it during the code challenge too lol